pkg/controller/cache/managed_test.go

```diff
@@ -277,10 +280,10 @@ func TestObserve(t *testing.T) {
 				mg: instance(
 					withConditions(runtimev1alpha1.Available()),
 					withBindingPhase(runtimev1alpha1.BindingPhaseUnbound),
-					withState(v1alpha2.StateReady),
-					withEndpoint(host),
+					withState(cloudmemorystore.StateReady),
```

Comment:
I just approved the PR but subsequently noticed this test (ObservedInstanceAvailable) is panicing because it's trying to call `Update` on a Kubernetes client that doesn't exist.